docs/product/accounts/migration/index.mdx

```diff
@@ -16,65 +16,72 @@ For additional reading on considering SaaS, take a look at:
+![Relocation getting started page](./img/relocation-landing-page.png)
 
-Run the following command in your terminal to start the backup script and export your data:
+**To receive a $100 credit, make sure to use the promo code `relocate100` on this page.**
 
-    $ ./scripts/backup.sh
+### 2. Choose which organizations to relocate
 
-This invokes the sentry CLI `export` command on a new isolated instance of our Docker web container.
+Your self-hosted installation may have had multiple organizations, each identified by a unique slug. You'll need to collect those from your installed instance, and decide which ones you'll want to keep. You can always change these to whatever you like post-relocation on the organization settings page.
 
-The export migrates your account's settings and configurations, including:
+![Changing the org slug on the settings page](./img/org-slug.png)
 
-- Project settings
-- Alert rules
-- Enabled data scrubbers
-- Team settings
-- Saved search queries
-- Integration platform metadata (Sentry 10)
+### 3. Follow the relocation instructions
 
-### 4. Send it to us
+The remaining pages in the relocation instructions will contain specific details on how to export your self-hosted instance's relocatable data. This will require terminal access to the machine running your self-hosted instance. In brief, you'll save a public key that we generate for you on your self-hosted machine, which you'll then use to execute a command that we provide to generate an encrypted tarball containing your data. You'll then upload this tarball, and we'll take it from there!
 
-To kick off the migration process, contact us using the form at [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/). Once we get your request, one of our support reps will reach out through a support ticket to get the following details:
+Most relocations complete in under an hour, though certain large or complex cases can take a bit longer. If you don't get an update from us within 24 hours, please [contact support](https://sentry.io/contact/enterprise/) for further assistance.
+
+### 4. Choose a plan
+
+Once your relocation succeeds, you can take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Note that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+
+If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, and single-tenant options, for example), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
```

Comment:
```suggestion
Once your relocation is complete, you can choose the [pricing plan](https://sentry.io/pricing/) that best suits your needs. Note that plan pricing varies based on your [Event Volume](/product/accounts/quotas/).

If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, and single-tenant options, for example), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
```
I think it's better to send them to our quota management docs in case they're not familiar with their expected monthly usage :)